lib/improver/tests/utilities/test_temporal.py

```diff
@@ -339,15 +211,15 @@ def test_seconds(self):
 
 class Test_datetime_constraint(IrisTest):
     """
-    Test construction of an iris.Constraint from a python.datetime.datetime
+    Test construction of an iris.Constraint from a python.datetime
```

Comment:
Perhaps this should be `python datetime object`, rather than `python.datetime object`?